packages/manager/apps/pci-instances/src/data/hooks/instance/useInstances.ts

```diff
@@ -141,6 +141,13 @@ const getActionHrefByName = (
     return { path: `${projectUrl}/workflow/new`, isExternal: true };
   }
 
+  if (name === 'assign_floating_ip') {
+    return {
+      path: `${projectUrl}/public-ips/order?ipType=floating_ip&region=${region}&instance=${id}`,
```

Comment:
maybe we could use search params to build it and make it more readable and avoid format issues